### PR TITLE
Add “Resources” section in the Airflow website #233

### PR DESCRIPTION
**Pull request by gdivya6028**
Add “Resources” section in the Airflow website #233

image

### CODE REVIEW

**Comment by divyaguduru** [comment]
pre-commit issues fixed

**Comment by potiuk** [comment]
I am not sure if adding the book to resources is a good idea. Generally whatever we publish (except Ecosystem page) should be PMC "owned" and "controlled" and the book is something written independently (even if one of the authors is a PMC members) -sp I think the right place to mention the book  is "ecosystem" page.

The "promo materials" though is a very good idea to add.

**Comment by divyaguduru** [comment]
@potiuk i will work on the feedback .. could you please review sticky sidebar implementation for "Community" page as well. It is requested on the same issue #233 

- Added sticky sidebar on scroll only for desktop, doesn't reflect on mobile or tablet screens.

https://github.com/user-attachments/assets/1e2905e6-a413-4f54-b186-abf5f0d3976c

**Comment by potiuk** [comment]
I am not good on the web side of it so maybe someone else.will, but if it works I see no reason for us not.merging it

**Comment by divyaguduru** [comment]
- Resources section is added in Community page
- Apache Airflow book updated in Ecosystem page
- Sticky sidebar for navigation is added in Community page for desktop view


https://github.com/user-attachments/assets/5fe28d96-a9e5-46c1-885b-1494147629bd

File: landing-pages/site/content/en/ecosystem/_index.md [hunk 1]
```diff
@@ -31,6 +31,8 @@ If you would you like to be included on this page, please reach out to the [Apac
 
 [Apache Airflow: Complete Hands-On Beginner to Advanced Class](https://www.udemy.com/course/apache-airflow-course) by Alexandra Abbas on Udemy
 
+[Data Pipelines with Apache Airflow](https://www.manning.com/books/data-pipelines-with-apache-airflow) Apache Airflow Book by Bas P. Harenslak and Julian Rutger de Ruiter on Amazon
```
Comment (potiuk) [inline landing-pages/site/content/en/ecosystem/_index.md:34]: BTW. I believe there is a second edition of that book being written as we speak by different authors, so maybe leaving the details out about the authorship might be a good idea.

Comment (gdivya6028) [reply landing-pages/site/content/en/ecosystem/_index.md]: @potiuk I've removed the authorship details for now .

**Review by potiuk — APPROVED** [review]

**Comment by potiuk** [comment]
can someone more familiar with javascript/css etc. review it as well @pierrejeambrun @jscheffl @bbovenzi ?

**Comment by bbovenzi** [comment]
Could we undo the whitespace changes to html and css? It's hard to tell what actually changed.

**Review by jscheffl — APPROVED** [review]
I checked the diff w/o whitespace... 
image


Is the build testable in a different URL?
Actually this PR mixed a bit of sticky layout changes with adding a menu entry. Next time might be easier to separate.

From point of code view... seems reasonable.

**Comment by bbovenzi** [comment]
> I checked the diff w/o whitespace... image
> 
> Is the build testable in a different URL? Actually this PR mixed a bit of sticky layout changes with adding a menu entry. Next time might be easier to separate.
> 
> From point of code view... seems reasonable.

Oh I never noticed that option before.

This generally looks good to me. It would still be best not to contribute whitespace changes.

**Comment by gdivya6028** [comment]
@potiuk / @jscheffl / @bbovenzi, Could you let me know if there are any actions required from my end here ?

**Review by potiuk — APPROVED** [review]

**Comment by potiuk** [comment]
> @potiuk / @jscheffl / @bbovenzi, Could you let me know if there are any actions required from my end here ?

Removing the whitespace?

**Comment by jscheffl** [comment]
> > @potiuk / @jscheffl / @bbovenzi, Could you let me know if there are any actions required from my end here ?
> 
> Removing the whitespace changes?

I would not insist on it. Would be better but.... LGTM

**Review by bbovenzi — APPROVED** [review]

**Pull request merged**